completions/bind

```diff
@@ -1,16 +1,73 @@
 # bash bind completion                                     -*- shell-script -*-
 
+_bind_var_vals()
+{
+    local pre=$1 var=$2 val=$3 vals
+    case $var in
+        bind-tty-special-chars|blink-matching-paren|byte-oriented|\
+        colored-completion-prefix|colored-stats|completion-ignore-case|\
+        completion-map-case|convert-meta|disable-completion|\
+        echo-control-characters|enable-bracketed-paste|enable-keypad|\
+        enable-meta-key|expand-tilde|history-preserve-point|\
+        horizontal-scroll-mode|input-meta|mark-directories|\
+        mark-modified-lines|mark-symlinked-directories|match-hidden-files|\
+        menu-complete-display-prefix|meta-flag|output-meta|page-completions|\
+        prefer-visible-bell|print-completions-horizontally|\
+        revert-all-at-newline|show-all-if-ambiguous|show-all-if-unmodified|\
+        show-mode-in-prompt|skip-completed-text|visible-stats)
+            vals=(on off)
+            ;;
+        bell-style)
+            vals=(audible none visible)
+            ;;
+        editing-mode)
+            vals=(emacs vi)
+            ;;
+        keymap)
+            vals=(emacs emacs-standard emacs-meta emacs-ctlx vi vi-command
+                vi-insert)
+            ;;
+        *)
+            return 1
+            ;;
+    esac
+    local v reset=$(shopt -p nocasematch); shopt -s nocasematch
+    for v in "${vals[@]}"; do
+        [[ $v == "$val"* ]] && COMPREPLY+=("${pre}${val}${v/#$val}")
+    done
+    eval "$reset"
+}
+
+_bind_vars()
+{
+    local pre=$1 var=$2
+    local v reset=$(shopt -p nocasematch); shopt -s nocasematch
+    while IFS=' ' read _ v _; do
+        [[ $v == "$var"* ]] && COMPREPLY+=("${pre}${var}${v/#$var}")
+    done < <(bind -v)
```

Comment:
Process substitution doesn't work under posix mode, see commit 153d6d4fab1faafbd33fccb0ba8a99964f05a95b and #190 for why simply unsetting it is not a good solution, need to come up with some other way.

Reply:
Got it. Replaced with a command substitution approach.